server/simple_server.go

```diff
@@ -306,16 +304,15 @@ func (s *SimpleServer) Register(svcI Service) error {
 		// register all context endpoints with our wrapper
 		for path, epMethods := range mcs.JSONEndpoints() {
 			for method, ep := range epMethods {
-				endpointName := metricName(prefix, path, method)
+				fullPath := strings.TrimPrefix(prefix+path, "/")
```

Comment:
Dunno, there's already a lot of repetition in the `Register()` functions. (something I'd love to get rid of / improve if you have suggestions).

I don't really ned the prefix in `TimedAndCounted()` but I could pass it anyway and do the concatenate and trim in there. Thoughts?